packages/braid-design-system/playroom/components.ts

```diff
@@ -0,0 +1,15 @@
+import dedent from 'dedent';
+
+declare const global: any;
+if (global?.__IS_PLAYROOM_ENVIRONMENT__ !== 'clearly') {
+  throw new Error(dedent`
+    Playroom components were imported instead of Braid components.
+    ‎
```

Comment:
There are invisible characters here to prevent VS Code from trimming unused white space. Because then `dedent` will collapse subsequent empty lines and the message will be too compacted.

<img width="634" alt="gremlins" src="https://user-images.githubusercontent.com/3297808/202619218-200d03a8-b95a-42bc-9fab-8ae630424bd6.png">

Without the ["gremlins"](https://marketplace.visualstudio.com/items?itemName=nhoizey.gremlins) it would look like this:

```
Playroom components were imported instead of Braid components.
Make sure to import from the correct entry point.
-import { Component } from 'braid-design-system/playroom/components';
+import { Component } from 'braid-design-system';
```